contributing.md

```diff
@@ -0,0 +1,153 @@
+    
+    The entry shows up on the troubleshooting pages for its associated products and topics. If the page doesn't already exist, add an entry for it in the page
+    index, setting `type` to `placeholder`. See [Navigation tree](#navigation-tree).
+
+#### Navigation tree
+
+The navigation hierarchy of a doc section is governed by `page-index/page-index.js` within the corresponding directory. To change the structure, for example, add or delete pages in a section, modify the corresponding `page-index.js`.
+
+An entry in a `page-index.js` includes the following fields: 
+
+|Key|Type|Required| Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|-|-|-|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|`href`|string|Yes| The URL segment to use for the page. If there is a corresponding Markdown file, `href` must match the name of the Markdown file, minus the file extension.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Can you use check marks and crosses instead of yes or no. Have a look a the table in https://docs-dev.timescale.com/docs-3452-docs-rfc-update-the-contribution-guide/self-hosted/3452-docs-rfc-update-the-contribution-guide/upgrades/minor-upgrade/#plan-your-upgrade-path.